subsystem/cluster_test.go

```diff
@@ -851,14 +851,18 @@ var _ = Describe("cluster install", func() {
 			Expect(swag.StringValue(c.Status)).Should(Equal(models.ClusterStatusInstalling))
 			Expect(swag.StringValue(c.StatusInfo)).Should(Equal("Installation in progress"))
 			Expect(len(c.Hosts)).Should(Equal(5))
+			var master *models.Host
 			for _, host := range c.Hosts {
 				Expect(swag.StringValue(host.Status)).Should(Equal(models.HostStatusInstalling))
+				if master == nil && host.Role == models.HostRoleMaster {
+					master = host
```

Comment:
@YuviGold @filanov Assigning a pointer to the iterator is not safe, right?

Reply:
good catch. I will just copy the ID

Reply:
@avishayt you are right - it will point to the last host.

Reply:
Good catch